client/state/stored-cards/assembler.js

```diff
@@ -0,0 +1,11 @@
+export const createStoredCard = card => {
```

Comment:
I would be interested to read that discussion actually. I have found only a few reducers which are using an assembler so I though something was going on here.
I wonder how we are handling moment's versions of date now for instance? is it always on the fly?